packages/bsp/common/usr/lib/armbian/armbian-live-patch

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+#
+
+SERVER_PATH="https://dl.armbian.com/_patch"
```

Comment:
Idea is that we use this in emergency situation, where we fu* up something and we can't resolve this usual apt way. I don't know if we need to complicate this more then this?

Patch should always read config [this](https://github.com/armbian/build/blob/main/packages/bsp/common/usr/lib/armbian/armbian-firstlogin#L9-L16) and then decide what to do.

Reply:
Alright. I am sure this will evolve as we start to use it in future. So no issues keeping it simple now. 

Just one last thing. Should we somehow sign and verify the patch? Because we are going to just download and execute code that user might not be aware of. If say an attacker poisons a user's dns or put a proxy, they can just send their own script and it will execute with root. So for security reasons, do we want to have a way to make sure we are not executing unknown code with root privileges?

Reply:
my bad. I see thats already covered. All good to go then

Reply:
>  If say an attacker poisons a user's dns or put a proxy, they can just send their own script and it will execute with root

If script is not signed with our key, it won't be executed.

> for security reasons, do we want to have a way to make sure we are not executing unknown code with root privileges?

Additional security measure here would be to verify if key /usr/share/keyrings/armbian.gpg has ID of DF00FAF1C577104B50BF1D0093D6889F9F0E78D5, even this is long shot. As attacker should place his key there ... If private key is compromised, we are doomed anyway.

Reply:
Added additional step. Perhaps this would do.